Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -2,5 +2,119 @@
 
 void trainingTask(uint8_t* data)
 {
+    loadMotorData(data[0], data[1]);
+    loadLightData(data[2]);
 
+    outputArray[0] = data[0];
+    outputArray[1] = data[1];
+    outputArray[2] = data[2];
+    validData = 0b00000111;
+
+    if(!checkMotors())
```

Comment:
You should rename the functions because when you read this, "if not checkMotors()" it doesn't make much sense. Something like `motorsValid()` or similar would be better.